ibl_to_nwb/AlyxToNWB/schema/example_metadata_output_file.json

```diff
@@ -489,8 +489,8 @@
       "description": "[nch] Each channel's row in its home file (look up via probes.rawFileName), counting from zero. Note some rows don't have a channel, for example if they were sync pulses"
     }
   ],
-  "Ecephys": {
-    "Ecephys": {
+  "ecephys": {
+    "ecephys": {
```

Comment:
no, the metadata forms have a capital E, the nwb file has a lowercase e

Reply:
can we not have the same convention in the metadata forms then?

Reply:
I've reverted these changes https://github.com/catalystneuro/IBL-to-nwb/pull/9/commits/982905feb4166b97fd924210663177fbf79deab7